### PR TITLE
(maint) Update library dependencies

### PR DESCRIPTION
**Pull request by kbarber**
Signed-off-by: Ken Barber <ken@bob.sh>

### CODE REVIEW

**Comment by mullr** [comment]
LGTM

File: project.clj [hunk 1]
```diff
@@ -9,10 +9,10 @@
    :password :env/nexus_jenkins_password
    :sign-releases false})
 
-(def tk-version "1.1.1")
-(def tk-jetty9-version "1.3.1")
-(def ks-version "1.1.0")
-(def tk-status-version "0.2.1")
+(def tk-version "1.2.0")
```
Comment (ajroetker) [inline project.clj]: This should be `1.3.0` now

File: project.clj [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
+(def tk-jetty9-version "1.5.0")
+(def ks-version "1.2.0")
```
Comment (ajroetker) [inline project.clj]: This should also be `1.3.0` now :)

File: project.clj (hunk 1 above)
Comment (mullr) [inline project.clj]: don't think we should upgrade to 1.3 until we're ready for HUP

Comment (kbarber) [inline project.clj]: Not only that, there are some dependency complexities when I do this that may or may not take time to resolve, I say lets hold off on it. This patch was about plucking the majority of the low hanging fruit, I think we can dive into the individual troublesome pieces that are left as separate work.

File: project.clj (hunk 2 above)
Comment (kbarber) [inline project.clj]: Tied to the above bump complexity, lets circle back on these bumps.

**Pull request merged**